pkg/sources/s3/s3.go

```diff
@@ -95,6 +95,10 @@ func (s *Source) Init(aCtx context.Context, name string, jobId sources.JobID, so
 
 	s.setMaxObjectSize(conn.GetMaxObjectSize())
 
+	if len(conn.Buckets) > 0 && len(conn.IgnoreBuckets) > 0 {
+		return fmt.Errorf("either a bucket include list or a bucket ignore list can be specified, but not both")
+	}
+
```

Comment:
I'll take a look at how hard this will be to enforce on the frontend, worst case I'll just add validation to the backend that gives the ol red text error message saying you can't have both

Reply:
oh i assumed it was

Reply:
My understanding was we just preferred one over the other instead of returning an error, at least I think that's how Github and Gitlab work 🤔

Reply:
@ahrav that's because those support globbing. This source doesn't (and I don't think the effort to add it is worth it right now) so it doesn't make sense to use both.

Reply:
fair enough 👍